WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -381,12 +381,15 @@ class AztecPostViewController: UIViewController, PostEditor {
     /// Media Progress Coordinator
     ///
     fileprivate lazy var mediaProgressCoordinator: MediaProgressCoordinator = {
```

Comment:
As mentioned above, I think we can get away with removing this entirely, and we'll just access things through methods on the media coordinator.